pkg/repositories/transformers/execution.go

```diff
@@ -135,6 +136,10 @@ func UpdateExecutionModelState(
 				},
 			},
 		}
+	} else if request.Event.GetError() != nil {
```

Comment:
nit: I think this should move to the next if block

Reply:
done